sdk/inc/azure/core/az_mqtt5_connection_config.h

```diff
@@ -14,40 +14,41 @@
 
 #include <azure/core/az_config.h>
 
-#ifndef AZ_MQTT5_CONNECTION_DEFAULT_MIN_RETRY_DELAY_MSEC
+#ifndef AZ_MQTT5_CONNECTION_MIN_RETRY_DELAY_MSEC
 /**
- * @brief The default minimum retry delay in milliseconds.
+ * @brief The minimum delay in milliseconds between retry attempts to connect to the MQTT 5 broker.
  */
-#define AZ_MQTT5_CONNECTION_DEFAULT_MIN_RETRY_DELAY_MSEC (1000)
-#endif // AZ_MQTT5_CONNECTION_DEFAULT_MIN_RETRY_DELAY_MSEC
+#define AZ_MQTT5_CONNECTION_MIN_RETRY_DELAY_MSEC (1000)
+#endif // AZ_MQTT5_CONNECTION_MIN_RETRY_DELAY_MSEC
 
-#ifndef AZ_MQTT5_CONNECTION_DEFAULT_MAX_RETRY_DELAY_MSEC
+#ifndef AZ_MQTT5_CONNECTION_MAX_RETRY_DELAY_MSEC
 /**
- * @brief The default maximum retry delay in milliseconds.
+ * @brief The maximum delay in milliseconds between retry attempts to connect to the MQTT 5 broker.
  */
-#define AZ_MQTT5_CONNECTION_DEFAULT_MAX_RETRY_DELAY_MSEC (100000)
-#endif // AZ_MQTT5_CONNECTION_DEFAULT_MAX_RETRY_DELAY_MSEC
+#define AZ_MQTT5_CONNECTION_MAX_RETRY_DELAY_MSEC (100000)
+#endif // AZ_MQTT5_CONNECTION_MAX_RETRY_DELAY_MSEC
 
-#ifndef AZ_MQTT5_CONNECTION_DEFAULT_MAX_RANDOM_JITTER_MSEC
+#ifndef AZ_MQTT5_CONNECTION_MAX_RANDOM_JITTER_MSEC
 /**
- * @brief The default maximum random jitter in milliseconds.
+ * @brief The maximum delay in milliseconds to add to the delay between retry attempts to connect
+ * to the MQTT 5 broker.
  */
-#define AZ_MQTT5_CONNECTION_DEFAULT_MAX_RANDOM_JITTER_MSEC (5000)
-#endif // AZ_MQTT5_CONNECTION_DEFAULT_MAX_RANDOM_JITTER_MSEC
+#define AZ_MQTT5_CONNECTION_MAX_RANDOM_JITTER_MSEC (5000)
+#endif // AZ_MQTT5_CONNECTION_MAX_RANDOM_JITTER_MSEC
 
-#ifndef AZ_MQTT5_CONNECTION_DEFAULT_MAX_CONNECT_ATTEMPTS
+#ifndef AZ_MQTT5_CONNECTION_MAX_CONNECT_ATTEMPTS
 /**
- * @brief The default maximum number of connect attempts.
+ * @brief The maximum number of retry attempts to connect to the MQTT 5 broker.
```

Comment:
```suggestion
 * @brief The maximum number of retry attempts to connect to the MQTT 5 broker. No maximum number of retry attempts if set to -1.
```
(or something similar)